Mathlib/AlgebraicTopology/SimplicialSet/Basic.lean

```diff
@@ -455,4 +455,59 @@ noncomputable def standardSimplex : SimplexCategory ⥤ SSet.Augmented.{u} where
 
 end Augmented
 
+section applications
+variable {S : SSet}
```

Comment:
```suggestion
variable {S : SSet} {n : ℕ}
```
I think we can drop all the `{n}` from the statements. I saw the same pattern was already being used in the other file. I'm not sure why we're doing it, because I don't think it helps readability.
What do you think?